frontend/packages/ux-editor/src/components/Properties/PropertiesHeader/EditLayoutSetForSubform/EditLayoutSet/CreateNewSubformSection/SubformDataModel.test.tsx

```diff
@@ -71,6 +71,28 @@ describe('SubformDataModel', () => {
 
     expect(dataModelInput).toBeInTheDocument();
   });
+
+  it('Should display ErrorMessage when nameError is present', async () => {
+    const user = userEvent.setup();
+    renderSubformDataModelSelect({ displayDataModelInput: true });
+    const dataModelInput = screen.getByRole('textbox', {
+      name: textMock('ux_editor.component_properties.subform.create_new_data_model_label'),
+    });
+    await user.type(dataModelInput, 'new');
+    const errorMessage = screen.getByText(textMock('schema_editor.error_reserved_keyword'));
+    expect(errorMessage).toBeInTheDocument();
+  });
+
+  it('Should not display ErrorMessage when nameError is not present', async () => {
+    const user = userEvent.setup();
+    renderSubformDataModelSelect({ displayDataModelInput: true });
+    const dataModelInput = screen.getByRole('textbox', {
+      name: textMock('ux_editor.component_properties.subform.create_new_data_model_label'),
+    });
+    await user.type(dataModelInput, 'newName');
+    const errorMessage = screen.queryByText(textMock('schema_editor.error_reserved_keyword'));
+    expect(errorMessage).not.toBeInTheDocument();
+  });
```

Comment:
Looks great! Just a small nitpick: Could we merge these two tests into a single 'toggle' test? Start by verifying that `await user.type(dataModelInput, 'new');` displays an error, then edit the input to make it valid and confirm the error disappears.

Reply:
Fixed